.cp2k/Makefile

```diff
@@ -183,24 +180,24 @@ LIBSMM_ACC_ABS_DIR := $(shell find $(SRCDIR) -type d -name "libsmm_acc")
 
 ALL_PKG_FILES := $(shell find $(SRCDIR) -name "PACKAGE")
 OBJ_SRC_FILES  = $(shell cd $(SRCDIR); find . ! -name "dbcsr_api_c.F" ! -name "dbcsr_tensor_api_c.F" -name "*.F")
-OBJ_SRC_FILES += $(shell cd $(SRCDIR); find . -name "*.c")
+OBJ_SRC_FILES += $(shell cd $(SRCDIR); find . -path ../src/acc/opencl -prune -type f -name "*.c")
 
 # if compiling with GPU acceleration
 ifneq ($(ACC),)
-  # if compiling with nvcc
-  ifneq (,$(findstring nvcc,$(ACC)))
-    # All *.cpp files belong to the accelerator backend common between CUDA/HIP
-    OBJ_SRC_FILES += $(shell cd $(SRCDIR); find . -type f ! -name "acc_cuda.cpp" ! -name "acc_hip.cpp" -name "*.cpp")
-    OBJ_SRC_FILES += $(LIBSMM_ACC_DIR)/../cuda/acc_cuda.cpp
-    # Exclude autotuning files
-    OBJ_SRC_FILES += $(shell cd $(SRCDIR); find . -type f ! -name "tune_*_exe*_part*.cu" ! -name "tune_*_exe*_main*.cu"  -name "*.cu")
-  # if compiling with hipcc
-  else ifneq (,$(findstring hipcc,$(ACC)))
-    # All *.cpp files belong to the accelerator backend common between CUDA/HIP
-    OBJ_SRC_FILES += $(shell cd $(SRCDIR); find . -type f ! -name "acc_cuda.cpp" ! -name "acc_hip.cpp" -name "*.cpp")
-    OBJ_SRC_FILES += $(LIBSMM_ACC_DIR)/../hip/acc_hip.cpp
-  # OpenCL backend: OBJ_SRC_FILES already includes all *.c files
-  endif
+	ifeq (cuda,$(USE_ACCEL))
```

Comment:
I recommend removing tabs like https://github.com/hfp/dbcsr/commit/193e77555ce94ea6da3940f9602b3aea986484b5. Using tabs outside of rules for indentation purpose can cause problems depending on the version/verbosity of GNU Make.

Reply:
This is done. Please check.

Reply:
Looks good to me, Thank you!